internal/service/cloudbackupsnapshotrestorejob/data_source_cloud_backup_snapshot_restore_job.go

```diff
@@ -103,17 +89,7 @@ func DataSource() *schema.Resource {
 func dataSourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
 	conn := meta.(*config.MongoDBClient).AtlasV2
 
-	var restoreID string
-	restoreIDRaw, restoreIDInField := d.GetOk("snapshot_restore_job_id")
-	if restoreIDInField {
-		restoreID = restoreIDRaw.(string)
-	} else {
-		idEncoded, restoreIDInField := d.GetOk("job_id")
-		if !restoreIDInField {
-			return diag.Errorf("either snapshot_restore_job_id or job_id must be set")
-		}
-		restoreID = conversion.GetEncodedID(idEncoded.(string), "snapshot_restore_job_id")
-	}
+	restoreID := d.Get("snapshot_restore_job_id").(string)
```

Comment:
@EspenAlbert would be good if you can take an extra look on these changes.

Reply:
Looks good 👍